content/en/blog/_posts/2022-12-06-kubernetes-1.25-blog.md

```diff
@@ -0,0 +1,205 @@
+
+### Improvements to Kubernetes metrics
+
+This release has several noteworthy improvements on metrics.
+
+#### Metrics framework extension graduates to Alpha
+
+The metrics framework extension [graduates to Alpha](https://github.com/kubernetes/enhancements/issues/3498), and [documentation](/docs/reference/instrumentation/metrics/) is now published for every metric in the Kubernetes codebase. This enhancement adds two additional metadata fields to Kubernetes metrics: `Internal` and `Beta`, representing different stages of metric maturity.
+
+#### Component Health Service Level Indicators graduates to Alpha
+
+Also improving on the ability to consume Kubernetes metrics, [component health Service Level Indicators (SLIs)](https://github.com/kubernetes/enhancements/issues/3466) have [graduated to Alpha](https://github.com/kubernetes/kubernetes/pull/112884): by enabling the `ComponentSLIs` feature flag there will be an additional metrics endpoint which allows the calculation of Service Level Objectives (SLOs) from raw healthcheck data converted into metric format.
+
+#### Feature metrics are now available
+
+Feature metrics are now available for each Kubernetes component, making it possible to [track whether each active feature gate is enabled](https://github.com/kubernetes/kubernetes/pull/112690) by checking the component's metric endpoint for `kubernetes_feature_enabled`.
+
+### Dynamic Resource Allocation graduates to Alpha
+
+[Dynamic Resource Allocation](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/3063-dynamic-resource-allocation/README.md) is a new feature that puts resource scheduling in the hands of 3rd-party developers: it moves away from the limited "countable" interface for requesting access to resources (e.g. `nvidia.com/gpu: 2`), providing an API more akin to that of persistent volumes. Under the hood, it uses the [Container Device Interface](https://github.com/container-orchestrated-devices/container-device-interface) (CDI) to do its device injection. This feature is blocked by the `DynamicResourceAllocation` feature gate.
+
+### CEL in Admission Control graduates to Alpha
+
+This feature introduces a `v1alpha1` API for [validating admission policies](https://github.com/kubernetes/enhancements/issues/3488), enabling extensible admission control via [Common Expression Language](https://github.com/google/cel-spec) expressions. Currently, custom policies are enforced via [admission webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/), which, while flexible, have a few drawbacks when compared to in-process policy enforcement. To use, enable the `ValidatingAdmissionPolicy` feature gate and the `admissionregistration.k8s.io/v1alpha1` API via `--runtime-config`.
```

Comment:
Features don't exactly **graduate** to alpha; instead, they arrive as a _new_ alpha feature.